src/gui/wizard/owncloudwizard.cpp

```diff
@@ -245,4 +246,22 @@ AbstractCredentials *OwncloudWizard::getCredentials() const
     return 0;
 }
 
+bool OwncloudWizard::askExperimentalPlaceholderFeature()
```

Comment:
Can we do it without re-entering the eventloop? Have a callback in a std::function<void(bool)>.